docs/FRSKY_R9.md

```diff
@@ -21,31 +21,45 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
 
-2. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to repower the module to make any change effective.
+3. If you have the older 2018 "ACCST" version of the R9M, you will want to perform the Inverter Mod to allow reliable communication with the radio at higher bit rates.  This is nicely documented in the [ExpressLRS documentation](https://www.expresslrs.org/hardware/inverter-mod/).
 
 ### DIY Inverter Dongle ###
 
-There are several DIY apporaches for building the inverter dongle. A common approach is based on the MAX3232 RS232 chip; an excellent build tutorial is provided here [Some soldering required](https://discuss.ardupilot.org/t/some-soldering-required/27613). Be aware however that many fake chips are available, and the original scheme suggested in this blog may not work. One in fact may have to add an extra diode and resistor.
+There are several DIY approaches for building an inverter dongle. A common approach is based on the MAX3232 RS232 chip; an excellent build tutorial is provided here [Some soldering required](https://discuss.ardupilot.org/t/some-soldering-required/27613). Be aware however that many fake chips are available, and the original scheme suggested in this blog may not work. One, in fact, may have to add an extra diode and resistor.
 
 <img src="images/frsky-max3232-inverter-scheme.jpg" width="360px">
 
+### ESP32 WiFi Bridge ###
+
+The mLRS git repository includes source code in the esp folder which allows several supported ESP32 development boards to be used as a WiFi Bridge to connect the serial port to a GCS such as Mission Planner, QGroundControl, APM Planner 2.0, or MAVProxy.  This approach can also eliminate the need for a separate inverter circuit.  Two of these boards, the M5Stamp Pico Mate and the M5Stack M5Stamp C3U Mate allow pin layouts which are especially convenient to connect directly to the serial pins on the back of the R9M Tx module.  The [M5Stack M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it's flash can be programmed via it's included USB port rather than requiring a separate programmer.  The 2.4GHz WiFi bridge works especially well with 900MHz radios like the R9M since the separate frequency range minimizes interference.
+
+To install the the sketch on the M5Stamp C3U, use the Arduino IDE;  Open the mlrs-wifi-bridge sketch, edit the mlrs-wifi-bridge.ino file and uncomment just the MODULE\_M5STAMP\_C3U\_MATE\_FOR\_FRSKY\_R9M define, select the ESP32C3 Dev board in the IDE, Connect the M5Stamp C3U module USB connector to your computer while holding down the center button, and upload the sketch via the IDE.
```

Comment:
- shouldn't it be a . instead of a ;
- maybe add .ino to be clear
- here we could refer to the esp folder, like "Open the mlr-sifi-bridge.ino sketch in the esp folder, ..."
- "connect" not "Connect"